Common/Systems/ModPlayers/UniversalBuffingPlayer.cs

```diff
@@ -1,10 +1,13 @@
 ﻿using System.Collections.Generic;
+using System.Linq;
+using PathOfTerraria.Common.Systems.Affixes;
 using PathOfTerraria.Core.Items;
 
 namespace PathOfTerraria.Common.Systems.ModPlayers;
 internal class UniversalBuffingPlayer : ModPlayer
 {
 	public EntityModifier UniversalModifier;
+	public AffixTooltipsHandler AffixTooltipHandler = new();
```

Comment:
Not really required in this scenario, but we might want to watch out to unloading resources whenever we can by clearing and nulling collections, etc.